ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -3770,7 +3770,7 @@ private static Path createDummyFileForEmptyTable(JobConf job, MapWork work,
   /**
    * setInputPaths add all the paths in the provided list to the Job conf object
    * as input paths for the job.
-   *
+   * 
```

Comment:
Unrelated change.

Reply:
> 

I will remove it since I add it for triggering CI